DEPS

```diff
@@ -22,6 +22,8 @@ vars = {
   'shaderc_revision': '65adcb504d024bb29f0ad8279b6a215d5f679832',
   'jsoncpp_revision': '571788934b5ee8643d53e5d054534abbe6006168',
   'jsoncpp_source_revision': '645250b6690785be60ab6780ce4b58698d884d11',
+  'vulkan-headers_revision':'5b44df19e040fca0048ab30c553a8c2d2cb9623e',
+  'vulkan-validation-layers_revision':'9fba37afae13a11bd49ae942bf82e5bf1098e381',

```

Comment:
If we don't need Vulkan validation layers in Aquarium, just need override `dawn_enable_vulkan_validation_layers` with false in .gn. Not need add these dependencies.

Reply:
Vulkan validation layer is quite useful for Aquarium to catch errors in dawn_vulkan backend.